README.md

```diff
@@ -1,2 +1,267 @@
-# Microverse_code-review
-Learning how to review code, a project from the Microverse curriculum
+<a name="readme-top"></a>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
- [ ] Well done, you did great but I would like to suggest trying to be more polite, clear and professional and consider using `please`, and `kindly` words in your comment and also tell how can comment useful or how a person makes a good, necessary or relevant and information comments.

### Your Comment

> I suggest you consider removing unnecessary comments in your codebase. This will allow you to have a clean code.